File: examples/android/CMeSdkJava/app/src/main/java/me/connect/sdk/java/sample/db/entity/ProofRequest.java
package me.connect.sdk.java.sample.db.entity;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity
public class ProofRequest {
    @PrimaryKey(autoGenerate = true)
    public int id;

    @ColumnInfo(name="thread_id")
    public String threadId;

    @ColumnInfo(name = "pwdid")
    public String pwDid;

    @ColumnInfo(name = "serialized")
    public String serialized;

    @ColumnInfo(name = "attachConnection")
    public String attachConnection;

    @ColumnInfo(name = "attachConnectionName")
    public String attachConnectionName;

    @ColumnInfo(name = "attachConnectionLogo")
    public String attachConnectionLogo;
}

File: examples/android/CMeSdkJava/app_kt/src/main/kotlin/me/connect/sdk/java/samplekt/db/entity/ProofRequest.kt
package me.connect.sdk.java.samplekt.db.entity

import androidx.room.ColumnInfo
import androidx.room.Entity

import androidx.room.PrimaryKey


@Entity
data class ProofRequest(
        @PrimaryKey(autoGenerate = true)
        var id: Int = 0,

        @ColumnInfo(name = "thread_id")
        var threadId: String,

        @ColumnInfo(name = "pwdid")
        var pwDid: String? = null,

        @ColumnInfo(name = "serialized")
        var serialized: String,

        @ColumnInfo(name = "name")
        var name: String,

        @ColumnInfo(name = "attributes")
        var attributes: String,

        @ColumnInfo(name = "accepted")
        var accepted: Boolean? = null,

        @ColumnInfo(name = "message_id")
        var messageId: String? = null,

        @ColumnInfo(name = "attachConnection")
        var attachConnection: String? = null,

        @ColumnInfo(name = "attachConnectionName")
        var attachConnectionName: String? = null,

        @ColumnInfo(name = "attachConnectionLogo")
        var attachConnectionLogo: String? = null
)